src/modules/auth/auth.service.ts

```diff
@@ -82,4 +89,54 @@ export class AuthService {
       role,
     };
   }
+
+  async forgotPassword({ email }: AuthForgotPasswordDto) {
+    const user = await this.authRepo.findOneBy({ email });
+    console.log(user);
+
+    if (user) {
+      const userAbout = await this.signUser(user.id, user.email);
+
+      const result = await this.forgotPasswordRepo.save({
+        user: user,
+        link: userAbout.token,
+      });
+
+      console.log(result);
+
+      const url = `http://localhost:3000/restore-password?token=${userAbout.token}`;
```

Comment:
never hardcode url in the code. move it to env or to constants

Reply:
done